schema_registry.go

```diff
@@ -18,11 +18,13 @@ package go_kafka_client
 import (
 	"encoding/json"
 	"fmt"
-	avro "github.com/stealthly/go-avro"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"strconv"
 	"strings"
+
+	avro "github.com/stealthly/go-avro"
```

Comment:
the above library should support the confluent schema registry, I know a lot of that support is in this library but I feel it makes sense to move to go-avro, yes? Also some folks might want to use the linked in avro repo (I know two folks that want to-do that) so we should support either library please. Both of these are seperete to-dos not necessarly for this branch to merge blockers but next iteration (or the one after).